### PR TITLE
fix(clerk-js): Correct initial ref value for form feedback

### PR DESCRIPTION
**Pull request by desiprisg**
## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerkinc/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: .changeset/breezy-bikes-grin.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (anagstef) [inline .changeset/breezy-bikes-grin.md:1]: Please add an entry to the changeset

**Review by anagstef — CHANGES_REQUESTED** [review]
Just add a changeset

**Review by anagstef — APPROVED** [review]

**Review by octoper — APPROVED** [review]

**Comment by dimkl** [comment]
Closing this one as we have pushed the commit in the main-v4

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request closed without merging**